### ISSUE TITLE
调用skin_retouching = pipeline(Tasks.skin_retouching,  model='./damo/model/cv_unet_skin-retouching'), skin_retouching("image")  该python进程 占用cpu RES 缓慢增加 ，OOM killer

### ISSUE THREAD

**Issue by hanziying** [modelscope/modelscope#787]
Thanks for your error report and we appreciate it a lot.

**Checklist**

* I have searched the tutorial on modelscope  [doc-site](https://modelscope.cn/docs)
* I have searched related issues but cannot get the expected help.
* The bug has not been fixed in the latest version.

**Describe the bug**

**To Reproduce**
* What command or script did you run?
>  A placeholder for the command.
* Did you make any modifications on the code or config? Did you understand what you have modified?
* What dataset did you use?

**Your Environments (__required__)**
* OS: `uname -a`
* CPU: `lscpu`
* Commit id (e.g. `a3ffc7d8`)
* You may add addition that may be helpful for locating the problem, such as
    * How you installed PyTorch [e.g., pip, conda, source]
    * Other environment variables that may be related (such as $PATH, $LD_LIBRARY_PATH, $PYTHONPATH, etc.)

Please @ corresponding people according to your problem:

Model related: @wenmengzhou @tastelikefeet

Model hub related: @liuyhwangyh

Dataset releated: @wangxingjun778

Finetune related: @tastelikefeet  @Jintao-Huang

Pipeline related: @Firmament-cyou @wenmengzhou

Contribute your model: @zzclynn

**Comment by hanziying** [comment]
%CPU   %MEM      COMMAND                                                             
         0.0       31.0          python
    
    接收并处理图像，运行一段时间差不多12小时 (和处理的图片量有关系)
    %CPU  %MEM     COMMAND                                                             
       0.0        66.0         python
    
    objgraph.show_growth(limit=50) 定位到
    patch_aggregation_overlap(image, h, w, padding=32):
         rearrange     操作会增加list、dict等对象增加
        
    放弃rearrange操作，自己采取其他方法实现后，objgraph.show_growth(limit=50)打印对象个数一直未见增加，但是cpu内存还在一直上升，没有解决问题
